app/controllers/mfa_confirmation_controller.rb

```diff
@@ -20,56 +20,18 @@ def skip
     redirect_to after_skip_path
   end
 
-  def new
```

Comment:
should we remove these from `routes.rb` too?

Reply:
oops, yep